### ISSUE TITLE
Xray Cloud: unable to create test execution due to missing summary

### ISSUE THREAD

**Issue by kconkas** [fundakol/behave-xray#11]
I have installed behave-xray from source, commit 41647779261d3318adf298e9036f5cd785c70145 and configured my feature files as described in README.md (namely added `@jira.test_plan` and `@jira.testcase`). When I try to run behave test with `behave_xray:XrayCloudFormatter` formatter, I get the following error:
```
EBUG:urllib3.connectionpool:https://xray.cloud.getxray.app:443 "POST /api/v2/import/execution HTTP/1.1" 400 155
ERROR:behave_xray.xray_publisher:HTTPError: Could not post to JIRA service at https://xray.cloud.getxray.app/api/v2/import/execution. Response status code: 400
Traceback (most recent call last):
  File "/Users/kristijan.conkas/.pyenv/versions/3.7.13/lib/python3.7/site-packages/behave_xray-0.0.0-py3.7.egg/behave_xray/xray_publisher.py", line 41, in publish_xray_results
    response.raise_for_status()
  File "/Users/kristijan.conkas/.pyenv/versions/3.7.13/lib/python3.7/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://xray.cloud.getxray.app/api/v2/import/execution
ERROR:behave_xray.xray_publisher:Error message from server: Error creating Test Execution - Issue create failed! - summary: You must specify a summary of the issue., description: Description is required.
ERROR:behave_xray.xray_publisher:Could not publish results to Jira XRAY
ERROR:behave_xray.xray_publisher:HTTPError: Could not post to JIRA service at https://xray.cloud.getxray.app/api/v2/import/execution. Response status code: 400
Error message from server: Error creating Test Execution - Issue create failed! - summary: You must specify a summary of the issue., description: Description is required.
```

**Comment by fundakol** [comment]
I can add summary and a description to a report. 
A description I can take from Feature's description:
```gherkin
Feature: feature name
  description
  further description
```
But I don't know what should be in a summary field. I can use some default text like `Execution of automated tests` or a text from an environment variable (that would be same text for all Features). 
Do you have any suggestions?

**Comment by kconkas** [comment]
> I can use some default text like Execution of automated tests or a text from an environment variable (that would be same text for all Features).

I think this would work. 👍  Ideally to take the text from an environment variable and default it to "Execution of automated tests" if the env. variable is missing or empty. What do you reckon?

**Comment by fundakol** [comment]
You can pass summary from command line or behave.ini file:
`$ behave -f behave_xray:XrayCloudFormatter -D xray.summary="Report generated by behave"`

**Comment by fundakol** [comment]
Released as 0.1.3. I hope there is no other mandatory fields.
